examples/test.yaml

```diff
@@ -0,0 +1,23 @@
+type: prompt-engine
+config:
+  model_config:
+    max_tokens: 1024
+  description_prefix: "Hello"
+  description_postfix: "!"
+  newline_operator: "\n"
+  input_prefix: ">"
+  input_postfix: ""
+  output_prefix: ""
+  output_postfix: ""
+description: "This is the description"
+examples:
+  - input: "Roam around Mars"
+    response: "This will be possible in a couple years"
+  - input: "Drive a car"
+    response: "This is possible after you get a learner drivers license"
+flow_reset_text: "This is the flow reset text"
```

Comment:
Just a note - it's good you're using snake case here. In Node, use dashes. YAML has no specific naming conventions, and usually assumes the conventions of the rest of the codebase

Reply:
What if we use the dashes approach here too, and have it same for node. This way we can create a language agnostic yaml file ?